api/src/owner.rs

```diff
@@ -80,17 +80,20 @@ impl Owner {
 
 	/// Trigger a validation of the chain state.
 	///
+	/// # Arguments
+	/// * `fast_validation` - if false verify the rangeproof associated with each unspent output and all kernels
```

Comment:
i'd prefer this is called something like assume_valid_rangeproofs_kernels,
to reflect its importance and potential for harm.